scripts/close_inactive.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+require 'octokit'
+
+REPOS = ["theforeman/foreman"]
+KEEP = 6 # months
+THRESHOLD = DateTime.now << KEEP
+COMMENT = <<EOC
+This PR has been inactive for #{KEEP} months, closing for now.
+Feel free to reopen when you return to it.
+Thank you for your contribution, @%s!
+EOC
+LABELS = ["Inactive"]
+
+c = Octokit::Client.new(:access_token => ENV['GITHUB_OAUTH_TOKEN'])
+REPOS.each do |repo|
+  c.pull_requests(repo).each do |pr|
+    number = pr[:number]
+    user = pr[:user][:login]
+    updated_at = pr[:updated_at].to_datetime
+    if updated_at < THRESHOLD
+      puts "Closing PR #{number} with latest update #{updated_at}"
+      #c.add_comment(repo, number, COMMENT % user)
+      #c.add_labels_to_an_issue(repo, number, LABELS)
```

Comment:
> > +      #c.add_labels_to_an_issue(repo, number, LABELS)
> 
> Did you therefore want to set the config options to false to begin with?

Yeah, well, I am keeping them to `true` now since I was able to test
this against theforeman/foreman and it works just fine.

Fixed all your comments above.

It now uses the search API.

More than that, it now closes the 5 oldest PRs every run (week) so we
don't get spammed if there is bunch of them during a night run.

-- 
Later,
 Lukas #lzap Zapletal